pkgs/tools/inputmethods/interception-tools/dual-function-keys.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitLab, pkgconfig, libyamlcpp,
+  libevdev }:
+
+stdenv.mkDerivation rec {
+  pname = "dual-function-keys";
+  version = "1.1.0";
+
+  src = fetchFromGitLab {
+    owner = "interception/linux/plugins";
+    repo = pname;
+    rev = version;
+    sha256 = "07hksca4g7v4zsvhmhc9j725j3n63fzrkx9sb4a0wrd7rwgr25rz";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ libevdev libyamlcpp ];
+
+  prePatch = ''
+    substituteInPlace config.mk --replace \
+      '/usr/include/libevdev-1.0' \
+      "$(pkg-config --cflags libevdev | cut -c 3-)"
+  '';
+
+  installPhase = ''
```

Comment:
What about `installFlags = [ "DESTDIR=$(out)" ];`?

Reply:
Thanks! I've made this change.